internal/provisioner/model/model.go

```diff
@@ -596,18 +604,10 @@ const (
 	ContourAvailableConditionType = "Available"
 )
 
-// OwningSelector returns a label selector using "projectcontour.io/owning-gateway-name".
-func OwningSelector(contour *Contour) *metav1.LabelSelector {
-	return &metav1.LabelSelector{
-		MatchLabels: map[string]string{
-			OwningGatewayNameLabel: contour.Name,
-		},
-	}
-}
-
 // OwnerLabels returns owner labels for the provided contour.
 func OwnerLabels(contour *Contour) map[string]string {
 	return map[string]string{
-		OwningGatewayNameLabel: contour.Name,
+		ContourOwningGatewayNameLabel:    contour.Name,
+		GatewayAPIOwningGatewayNameLabel: contour.Name,
```

Comment:
ref. https://github.com/projectcontour/contour/issues/5970

Reply:
ha, on this note we should probably vet the label values to make sure theyre <= 63 chars